.github/workflows/i18n-pull-base.yml

```
@@ -29,8 +29,11 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v6
         with:
-          commit-message: i18n:pulled translations
-          title: i18n:pulled translations
-          body: i18n:pulled translations
+          commit-message: |
+            i18n: pulled translations
```

```suggestion
          commit-message: "i18n: pulled translations"
```
wouldn't it also work by adding quotes around the string, to disable the weird YAML magic?